core/src/test/scala/org/apache/spark/JobCancellationSuite.scala

```diff
@@ -72,7 +73,7 @@ class JobCancellationSuite extends SparkFunSuite with Matchers with BeforeAndAft
   }
 
   test("cluster mode, FIFO scheduler") {
-    val conf = new SparkConf().set("spark.scheduler.mode", "FIFO")
+    val conf = new SparkConf().set(SCHEDULER_MODE, "FIFO")
```

Comment:
Do you want to use `SchedulingMode.FIFO.toString` in cases like this? no big deal, it doesn't matter much